extensions/wikia/CityList/CityList.setup.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Handle updating shared city_list table
+ */
+
+$GLOBALS['wgAutoloadClasses']['UpdateCityListTask'] = __DIR__ . '/UpdateCityListTask.php';
```

Comment:
`$GLOBALS` ?

Reply:
Yeah, as it is included in method scope in the test file - this way config changes do not break test